### PR TITLE
fix: set the proper hostname for staging contxt-coordinator

### PR DESCRIPTION
**Pull request by andreimoustache**

### CODE REVIEW

File: src/config/audiences.js [hunk 1]
```diff
@@ -28,7 +28,7 @@ export default {
     },
     staging: {
       clientId: '8qY2xJob1JAxhmVhIDLCNnGriTM9bct8',
```
Comment (branchvincent) [inline src/config/audiences.js:30]: i **think** this needs to be updated too:
```suggestion
      clientId: 'qGzdTXcmB57zlTp86rYsivG9qEss1lbF',
```

Comment (andreimoustache) [reply src/config/audiences.js:30]: Hmmm, any idea on how I could check that?

Comment (branchvincent) [reply src/config/audiences.js:30]: this should be right, i pulled this from the api itself: https://github.com/ndustrialio/contxt-api/blob/14c3199dbb082c81b041f48b9214fc7c9ce7fbc9/charts/contxt/values-staging.yaml#L59-L60

Comment (branchvincent) [reply src/config/audiences.js:30]: to verify, we can just grab a JWT for that audience and try to make a request. the only wrinkle here is it looks like that client only exists in the staging auth0 tenant: https://github.com/ndustrialio/contxt-api/blob/14c3199dbb082c81b041f48b9214fc7c9ce7fbc9/charts/contxt/values-staging.yaml#L86-L87

**Comment by branchvincent** [comment]
@andreimoustache releasing to NPM is manual right now, opened #231 to publish on merges to `main`

**Pull request merged**